platform/android/export/export.cpp

```diff
@@ -2455,6 +2465,15 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 
 		load_icon_refs(p_preset, main_image, foreground, background);
 
+		if (export_app_bundle == 1 && p_path.find(".aab") == -1) {
+			EditorNode::get_singleton()->show_warning(TTR("Cannot export App Bundle using a *.apk file name."));
```

Comment:
This error message is assuming that the user entered `.apk` while the extension could be missing or be something else.
Something like this would be better:
`Invalid filename! Android App Bundle requires the *.aab extension.`